main/options/options.cc

```diff
@@ -414,6 +414,8 @@ buildOptions(const vector<pipeline::semantic_extension::SemanticExtensionProvide
                                     cxxopts::value<string>()->default_value("autoloader"));
     options.add_options("advanced")("autogen-registry-module", "Name of Ruby module used for autoloader registry",
                                     cxxopts::value<string>()->default_value("Opus::Require"));
+    options.add_options("advanced")("autogen-root-object", "Name of Ruby object on which root autoloads should be installed",
+                                    cxxopts::value<string>()->default_value("Object"));
```

Comment:
I'm pretty sure this is true, but double checking, would this would support a root obj with leading `::`?

Reply:
Yup! All this does is stitch the string in without checking anything, so you could have qualified names or even a send here.